lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -59,7 +53,7 @@ def to_native(self, course):
             "video_outline": video_outline_url,
             "course_updates": course_updates_url,
             "course_handouts": course_handouts_url,
-            "course_about": course_about_url,
+            "course_about": "",
```

Comment:
Something to consider: removing course_about field from the endpoint's response entirely